src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -1,325 +1,205 @@
 ---
 title: Get started with synthetic monitoring
 tags:
-  - Synthetics
-  - Synthetic monitoring
-  - Getting started
+    - Synthetics
+    - Synthetic monitoring
+    - Getting started
+    - Synthetic monitoring tutorial
 translate:
-  - jp
+    - jp
 metaDescription: "New Relic's synthetic monitoring capabilities give you automated, scriptable tools to monitor your websites, critical business transactions, and API endpoints."
 redirects:
-  - /docs/new-relic-only/new-relic-synthetics/getting-started/new-relic-synthetics
-  - /docs/synthetics/new-relic-synthetics/getting-started/new-relic-synthetics
-  - /docs/synthetics/new-relic-synthetics/getting-started/introduction-new-relic-synthetics
-  - /docs/synthetics/synthetic-monitoring/getting-started/get-started-new-relic-synthetic-monitoring
-  - /docs/synthetics
-  - /docs/synthetics/synthetic-monitoring/getting-started
+    - /docs/new-relic-only/new-relic-synthetics/getting-started/new-relic-synthetics
+    - /docs/synthetics/new-relic-synthetics/getting-started/new-relic-synthetics
+    - /docs/synthetics/new-relic-synthetics/getting-started/introduction-new-relic-synthetics
+    - /docs/synthetics/synthetic-monitoring/getting-started/get-started-new-relic-synthetic-monitoring
+    - /docs/synthetics
+    - /docs/synthetics/synthetic-monitoring/getting-started
 ---
 
-You can think of our synthetic monitors as crash test dummies for your websites, applications, and API endpoints. When you deploy a synthetic monitor, an API check or virtual browser instance runs through your most important workflows, testing your critical endpoints and reporting the results to New Relic. If there's an error, failure, or anomaly affecting your customer-facing apps, then your synthetic monitor will catch it and alert you, all before anyone has any time to notice. 
+import syntheticAGifThatDemonstratesthePingMonitorWorkflow from 'images/synthetic_screenshot-full_A-gif-that-demonstrates-the-ping-monitor-workflow.gif'
 
-Our synthetic monitoring capabilities work on your internal apps and behind firewalls, too. Set up a private location, then send containerized applications to manage the jobs (that is, to deploy the kind of synthetic monitor you'd like to deploy) in a secured context. 
+import syntheticTest from 'images/testsyntheticsgif2.gif'
 
-## Synthetic monitors for proactive monitoring [#proactive]
+You can think of our synthetic monitors as crash test dummies for your websites, applications, and API endpoints. When you deploy a synthetic monitor, an API check or virtual browser instance runs through your most important workflows, testing your critical endpoints and reporting the results to New Relic. By mimicking the behaviors of customers, your monitors test how well your web apps handle traffic. 
```

Comment:
We shouldn't call this a virtual browser instance. We use real Chrome browsers to execute our browser monitor types.